applications/roloi.md

```diff
@@ -0,0 +1,161 @@
+
+6 years ago **we founded NeoPower**, our own software company to work for different clients building freelance teams of designers, developers and testers.
+
+We started our journey with NeoPower to solve real-world problems with Web3 products that are secure and have a great user experience. Roloi is our main product.
+
+
+#### CGO
+
+**Hernán S. Hermida** (aka [Milstein](https://twitter.com/MilsteinmAb)) is a **DeFi Researcher** with more than 2 years of experience contributing with educational communities in Latam. Currently, he **hosts #DeFiSpace** a weekly Twitter Spaces cycle, with more than 50 episodes, interviewing builders from different multi-chain projects, and creates content for [“La Multisig”](https://www.youtube.com/@LaMultisig), a web3 educational YouTube Channel with its own community. 
+
+He’s the growth lead of [DeFi Argentina](https://twitter.com/DeFiArgentina) a non-profit project that helps children’s food banks in Argentina by collecting donations in cryptocurrencies. 
+
+He’s also an **OAK ambassador** since Nov 2022.
+
+Hernán is part of the **Roloi team** as CGO to help with the growth, research and networking strategy, leveraging his knowledge about DeFi and communities.
+
+### Team Code Repos
+* [NeoPower Github Repository](https://github.com/NeoPower-Digital)
+
+## Development Status 
+
+We are working with the library [inkathon](https://github.com/scio-labs/inkathon) to simplify UI development and connection to the blockchain, this allows developers to focus more on solving actual problems.
+
+On the other hand we’ve been exploring OAK **code examples** and we identified some improvements that need to be built in the form of project templates to push forward the creation of new products using OAK.
+
+We had the opportunity to pitch Roloi to the OAK team and our plans to **build the core infrastructure needed to configure recurring transactions from our ink! smart contract.**
+
+## Development Roadmap 
+
+### Overview
+
+* **Total Estimated Duration:** 2 months
+* **Full-Time Equivalent (FTE):** 4
+* **Total Costs:** 50,000 USD
+
+### Milestone 1 — Automation UI Template
+
+* **Estimated duration:** 1 month
+* **FTE:** 4
+* **Costs:** 25,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | MIT |
+| 0b. | Documentation | The milestone will include technical documentation of the UI project. |
+| 0c. | Testing Guide | Core functions will be covered by unit tests to ensure functionality and robustness. We will describe how to run these tests in the docs. |
+| 1. | UI Source Code | We will deliver a Next.js UI project that enables the execution of a simple recurring task (on-chain event) and works as a template to be reused for any team that needs automated tasks. Tools and libraries: Next.js, Typescript, Material UI, XState. *The scope of this Milestone will cover only testnet chains.* |  
```

Comment:
1. By "a simple recurring task", do you refer to XCM remote execution via our `timeAutomation.scheduleXcmpTask`, or a task on Turing Network only?
2. My understanding is that you are aiming to automate !ink smart contract functions on Astar. Could we be more specific about the "a simple recurring task" and find a target smart contract function to call in the Rococo environment?
3. More importantly, if this is a UI project we need a mockup, or at least a wireframe to see what the product will look like. Please check out the wireframe in [our W3F Grant submission](https://github.com/w3f/Grants-Program/blob/master/applications/quadratic-funding.md) for reference.

Reply:
**1.** The goal of Milestone 1 is to enable the recurring execution of an on-chain event on Astar using `timeAutomation.scheduleXmpTask` or `timeAutomation.scheduleDynamicDispatchTask`.

The extrinsic to be executed would be something like this:
`api.tx.system.remarkWithEvent(message)`

We can improve this definition in our application file.

**2.** On Milestone 1 we will deliver a UI that works with recurring on-chain events. 
On Milestone 2 we will provide the functionality to execute recurring smart contract transactions. We will include a smart contract example to use as a default case.

On both milestones, the recurrence process will be done through the OAK `timeAutomation` API.

**3.** Perfect, we’ll work on wireframes and add them to the application.